src/components/atoms/menuItem/MenuItem.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../../foundations/sass/all";
+
+.menu-item {
+  // background-color: $white-floral;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
```

Comment:
Omitting the `align-items` property will not change the style.

Reply:
right